src/components/Quote.jsx

```diff
@@ -0,0 +1,109 @@
+import { useEffect, useState } from 'react';
+import fetchQuote from '../API/quoteApi';
+
+const Quote = () => {
+  const [data, setData] = useState({
+    quote: '',
+    author: '',
+    category: '',
+  });
+  const [status, setStatus] = useState(null);
```

Comment:
- [OPTIONAL] It's one of the project requirements you create a `loading` state and an `error` state that will handle errors. Please address this:

```jsx
  const [quotes, setQuotes] = useState([])
  const [hasError, setHasError] = useState(false)
  const [isLoading, setIsLoading] = useState(false)
```